modules/platforms/nodejs/package.json

```diff
@@ -32,8 +32,9 @@
     "test:auth_example": "node ./spec/ExamplesExecutor.js AuthExample"
   },
   "devDependencies": {
-    "jasmine": "latest",
+    "jasmine": "3.6.1",
```

Comment:
Should we used a fixed version for other dependencies below?

Reply:
I fixed other dev dependencies and decided to leave project dependencies as is.